src/2025h2/reflection-and-comptime.md

```diff
@@ -0,0 +1,131 @@
+# reflection and comptime
+
+| Metadata         |                                                                                  |
+|:-----------------|----------------------------------------------------------------------------------|
+| Point of contact | @oli-obk                                                                         |
+| Teams            | <!-- TEAMS WITH ASKS -->                                                         |
+| Task owners      | <!-- TASK OWNERS -->                                                             |
+| Status           | Proposed                                                                         |
+| Tracking issue   | https://github.com/rust-lang/rust/issues/142577                                  |
```

Comment:
IIUC "Tracking issue" is supposed to be the GH project goal issue where updates will be posted.

A r-l/r tracking issue instead could be:

- others have used the following

```suggestion
| Other tracking issues | [rust-lang/rust#142577] |
```

- the "See also" metadata field is also documented and supposedly used to track progress with checkboxes and so on -- but I don't know if anyone has used it / tested to see if it works 😓